preliz/tests/test_beta_mode.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+import numpy as np
+from preliz import beta_mode
+
+
+def test_beta_mode():
+
+    lower, upper, mode, mass, expected_alpha, expected_beta = 0.25, 0.75, 0.5, 0.9, 4.94, 4.94
```

Comment:
Just pass the values to the functions. Don't need to define this variables

Reply:
oh okay my bad. i have fixed it